components/button/test/button.html

```diff
@@ -0,0 +1,143 @@
+<!doctype html>
```

Comment:
Looking through the old button tests, we no longer seem to have tests for things we moved into the `ButtonMixin`. Things like testing that focusing works properly and that the `type` defaults to `button`. So maybe not in this PR, but we should add some tests for `ButtonMixin` in the future.